bundle/jsky.app.ot/src/main/java/jsky/app/ot/tpe/TelescopePosEditor.java

```diff
@@ -306,7 +309,11 @@ private boolean basePosUpdated(TpeContext oldCtx, TpeContext newCtx) {
         ITarget oldBase = oldBasePos.getTarget();
         ITarget newBase = newBasePos.getTarget();
 
-        return (oldBase.getRaDegrees() != newBase.getRaDegrees() || oldBase.getDecDegrees() != newBase.getDecDegrees());
+        Option<Long> oldWhen = oldCtx.schedulingBlockJava().map(SchedulingBlock::start);
+        Option<Long> newWhen = oldCtx.schedulingBlockJava().map(SchedulingBlock::start);
```

Comment:
Shouldn't  `newWhen` be obtained from `newCtx`? As it is looks like `oldWhen` and `newWhen` are the same thing. Is that the intention?